radixdlt/src/integration/java/com/radixdlt/consensus/simulation/synchronous/UniformLatencyTest.java

```diff
@@ -36,7 +36,7 @@
 	 */
 	@Test
 	public void given_4_correct_bfts__then_should_pass_sanity_tests_over_1_minute() {
```

Comment:
General comment on duration-based tests - it would be a good idea for us to have the duration defined in some config file rather than hard-coded in here. Sometimes I just want a quick sanity check without running 40 tests for a minute each. We can then have our CI use a higher duration to increase certainty (maybe even multiple minutes), while also having it be short and sweet (a few seconds) while developing.

Reply:
yeah totally agree